tethys_gizmos/static/tethys_gizmos/js/cesium_map_view.js

```diff
@@ -274,10 +274,18 @@ var CESIUM_MAP_VIEW = (function() {
         }
 
         for (let model of m_models) {
-            // options is where the model data is store.
-            let model_option = cesium_options(model.options);
+            var model_option;
+            // if model has options attribute, it should come from MVLayer.
+            if (model.options) {
+                // options is where the model data is store.
+                model_option = cesium_options(model.options);
+            }
+            else {
+            // else we will treat it as normal cesium object.
+                model_option = cesium_options(Object.values(model)[0]);
```

Comment:
Is `Object.values(model)[0]` necessary? Can you just pass it the model directly: `cesium_options(model)`. We don't do this for layers (see: line 240).

Reply:
It does because the MVLayer options and Object.values(model)[0] store the same data structure.